src/views/AutogenView.vue

```diff
@@ -726,6 +741,11 @@ export default {
           var hasJobId = false
           api(this.currentAction.api, params).then(json => {
             // set action data for reload tree-view
+            if (this.currentAction.api.startsWith('delete') ||
```

Comment:
I would prefer that we simply run the api of the action against all the selected items; the only exception could be if the parameter the API accepts is `ids` then instead of looping+calling on all items; we can simply call a single API. Let's discuss.

Reply:
@rhtyd  made the fix for looping+calling API for all selected items when `ids` parameter is not available.
Also, when `groupAction` is set to `true` and there is selection in list then we ignore show property of the action.